test/e2e/performanceprofile/functests/1_performance/cpu_management.go

```diff
@@ -751,8 +751,148 @@ var _ = Describe("[rfe_id:27363][performance] CPU Management", Ordered, func() {
 		})
 	})
 
+	Context("Runc cpus isolation", func() {
```

Comment:
This test case is not limited to runc,  since we execute across all container runtimes, you can rename this check container runtimes cpu usage

Reply:
we execute the test cases on both runtimes, so make sure that this test works on crun too.

Reply:
I verified the test execution on a crun runtime enabled cluster as well, It passed successfully.